src/store/editor-slice-utils.ts

```diff
@@ -371,6 +371,7 @@ export const generateRequest = (data: MailsEditor): SoapDraftMessageObj => {
 	}
 
 	return {
+		autoSendTime: data.autoSendTime ?? undefined,
```

Comment:
data?.autoSendTime is alreay enough and will return undefined if it is undefined